lib/system/caps_linux.go

```diff
@@ -91,5 +95,12 @@ func maxCap() (cap int, err error) {
 	return strconv.Atoi(strings.TrimSpace(string(buf[:n])))
 }
 
-// capSysChroot defines the number of the linux capability to execute chroot
-const capSysChroot = 18
+const (
+	// Capability to bypass file read, write, and execute permission checks.
+	// (DAC is an abbreviation of "discretionary access control".)
+	capDACOverride = 1
```

Comment:
Are there no constants already available somewhere for these caps we could use?

Reply:
Not that I know of - we have an integration with an external package in planet but not in gravity.